packages/nextjs-webpack-plugin/package.json

```diff
@@ -66,7 +66,7 @@
     "webpack": "^5.90.0"
   },
   "peerDependencies": {
-    "next": "14.x || 15.x",
+    "next": ">=14.2.10 <16.0.0",
```

Comment:
@suejung-sentry are we able to rollback to the prior implementation of peer deps declarations? 

With these changes, we'd have to move from a `patch` to a `major` release as we're requiring users to update their version of `next`. If they don't, peer dep resolution would fail while they try to install the plugin.

Reply:
Ah yeah good point. I won't force that upon people & just fix in our examples / integration-tests instead to resolve the vulnerability - thanks!